arbeitszeit_flask/api/plans.py

```diff
@@ -59,8 +59,9 @@ def get(
 class Plan(Resource):
     @namespace.marshal_with(plan_get_model, skip_none=True)
     @error_response_handling(
-        error_responses=[BadRequest, NotFound], namespace=namespace
+        error_responses=[BadRequest, NotFound, Unauthorized], namespace=namespace
```

Comment:
How is this list used, and why is it in this order?  It seems odd to me, because clients would have to refer to the error_responses element by index (or iterate over the list?), whereas I would expect the client to be clearer if it just specified, e.g., "Unauthorized."  BTW, what is the type of the elements in this list?